### PR TITLE
dataset .dvc files

### PR DESCRIPTION
**Pull request by SoyGema**
The final version contained:
**3 datasets** : train, test, answer . Therefore 3 .dvc files in its correspondent folder 
The datasets are stored remotely .

### CODE REVIEW

File: satellitedata/jan_test.csv.dvc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+outs:
+- md5: 4a8c9ac0a12753011e9692360fa92951
+  size: 22060747
+  path: jan_test.csv
```
Comment (shcheklein) [inline satellitedata/jan_test.csv.dvc:4]: qq - what does jan mean in this case? can it be just "test", "train"?

**Review by shcheklein — COMMENTED** [review]
Q (just to make sure) - do we need to have 3 files or it's easier to do a single directory?

**Comment by shcheklein** [comment]
let's please make a subdirectory - workshop or workshops. 

let's please add a brief README into subdirectory that explains where this dataset is being used, etc.

Thanks!

File: satellitedata/jan_test.csv.dvc (hunk 1 above)
Comment (SoyGema) [reply satellitedata/jan_test.csv.dvc:4]: **jan** makes reference to the month . January .

**Comment by SoyGema** [comment]
> Q (just to make sure) - do we need to have 3 files or it's easier to do a single directory?

Hey @shcheklein . You made a substantial question. 

In general, when facing the challenge of been given different datasets it is because the **patterns** or **statistical distributions** of training and testing can show differences that have impact on testing different model aspects, such as anomaly detection, overfitting and reactions to data drift. It might also ensure that the **number of samples** for each entity (in this case, Satellites) is tested.

Please, find this super quick exploration of how merging impacts, at least on train and test [sizes](https://github.com/iterative/VSCode-DVC-Workshop/blob/main/ExperimentsDVC/notebooks/Statistical%20Distribution%20Datasets.ipynb)


The **train_test_split** sklearn function doesn´t contemplate this. If this critera is deprecable in your opinion, let´s merge it.Let me know your thoughts and thanks!

**Comment by shcheklein** [comment]
@SoyGema my question was not about actual ML / DS implication based on statistics, etc. It's more about how do you use it - one single `dvc get ... data` vs multiple `dvc get`s - file by file.

**Comment by SoyGema** [comment]
> @SoyGema my question was not about actual ML / DS implication based on statistics, etc. It's more about how do you use it - one single `dvc get ... data` vs multiple `dvc get`s - file by file.

Makes sense. Ill do a single file. 
[Follow up](https://iterativeai.slack.com/archives/CB41NAL8H/p1669553828053349) with feedback from @dberenbaum and thumbs from @efiop about howto

**Comment by SoyGema** [comment]
Done
- [x]  Rename folder with data in AWS Remote to `workshop`. 
- [x]  Remove tracked 3 files with `dvc remove name.dvc `
- [x]  Rename folder to workshop
- [x] Add a unique `workshop.dvc` file for tracking 3 files

**Comment by jorgeorpinel** [comment]
Pls remove `.DS_Store` file 🙂 

> please add a brief README into subdirectory

Or maybe add a section to the [main one](https://github.com/iterative/dataset-registry/blob/master/README.md)?

File: .gitignore [hunk 2]
```diff
@@ -1 +0,0 @@
-.env/
```
Comment (shcheklein) [inline .gitignore:1]: no need to delete this file? this is not related to this PR

File: workshop/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/workshop
```
Comment (shcheklein) [inline workshop/.gitignore:1]: let's make it `workshop/<name of this specific workshop>` please

**Comment by shcheklein** [comment]
> Or maybe add a section to the [main one](https://github.com/iterative/dataset-registry/blob/master/README.md)?

this is less scalable I think, let's make it a readme per dataset

@SoyGema thanks, some minor things left! (I think it's not even that relevant for you anymore? sorry for all these delays)

File: workshop/.gitignore (hunk 3 above)
Comment (SoyGema) [reply workshop/.gitignore:1]: This workshop has been already selected for being repeated in other events across Python Spain Network .

If you are ok with this , we can follow the same mental model as we do for other datasets for  **consistency**  
 and future material **leveraging** , and do `workshop/satellitedata` . 

Let me know your thoughts :)

**Comment by SoyGema** [comment]
> > Or maybe add a section to the [main one](https://github.com/iterative/dataset-registry/blob/master/README.md)?
> 
> this is less scalable I think, let's make it a readme per dataset
> 
> @SoyGema thanks, some minor things left! (I think it's not even that relevant for you anymore? sorry for all these delays)

No worries at all. If you are ok with this, I think that is still relevant to ensure reproducibility, consistency across other examples , and preparation for future workshops. Thanks @jorgeorpinel for being part of this, I really appreciate it. 🥇

**Comment by jorgeorpinel** [comment]
I'm just nosy.

File: workshop/.gitignore (hunk 3 above)
Comment (shcheklein) [reply workshop/.gitignore:1]: yes, absolutely, `workshop/satellitedata` makes sense to

**Comment by SoyGema** [comment]
> Pls remove `.DS_Store` file 🙂
> 
> > please add a brief README into subdirectory
> 
> Or maybe add a section to the [main one](https://github.com/iterative/dataset-registry/blob/master/README.md)?

Done

**Comment by SoyGema** [comment]
- [x]  Remove .DS_Store file
- [x] Add README.md file
- [x] Merge changes from master with changes coming from @dberenbaum
